### PR TITLE
ci: Test with Python 3.13

### PR DESCRIPTION
**Pull request by edgarrmondragon**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -50,6 +58,7 @@ jobs:
         with:
           name: coverage-${{ github.job }}-${{ strategy.job-index }}
           path: .coverage.*
+          include-hidden-files: true
```
Comment (henryiii) [inline .github/workflows/ci.yml:61]:
```suggestion
          include-hidden-files: true
          if-no-files-found: error
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -39,6 +46,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
           miniforge-variant: Mambaforge
           use-mamba: true
+          channels: conda-forge/label/python_rc,conda-forge
```
Comment (edgarrmondragon) [inline .github/workflows/ci.yml:49]: Nice, TIL @henryiii!

Comment (henryiii) [reply .github/workflows/ci.yml:49]: I figured it out today too. :) I've been seeing my feedstocks being rebuilt for 3.13, so I knew it had to be somewhere.

**Comment by henryiii** [comment]
This also contains a fix for upload-artifact v4.4 now avoiding hidden files by default.

**Comment by henryiii** [comment]
Looks like the action tests are only running on main. Those also need 3.13.

**Pull request merged**